README.md

```diff
@@ -280,33 +280,51 @@ This also follows the [gitignore](https://git-scm.com/docs/gitignore) convention
 
 See [.wokeignore.example](.wokeignore.example) for a collection of common files and directories that may contain generated [SHA](https://en.wikipedia.org/wiki/Secure_Hash_Algorithms) and [GUID](https://en.wikipedia.org/wiki/Universally_unique_identifier)s. Dependency directories are also shown in the example as the linter will parse dependency source code and possibly find errors.
 
-#### In-line ignoring
+#### In-line and next-line ignoring
 
 There may be times where you don't want to ignore an entire file.
-You may ignore a specific line for one or more rules by creating an in-line comment.
+You may ignore a specific line for one or more rules by creating an in-line or next-line comment.
 
 This functionality is very rudimentary, it does a simple search for the phrase. Since
 `woke` is just a text file analyzer, it has no concept of the comment syntax for every file
 type it might encounter.
 
-Simply add the following to the line you wish to ignore, using comment syntax that is supported for your file type.
+For in-line ignoring, simply add the following to the line you wish to ignore, using comment syntax that is supported for your file type.
 (`woke` is not responsible for broken code due to in-line ignoring. Make sure you comment correctly!)
 
+Next-line ignoring works in a similar way. Instead of adding to the end of line you wish to ignore, you can create the ignore comment on its own line just before it. Any alphanumeric text to the left of the phrase will cause `woke` to treat it as an in-line ignore, but any text to the right of the phrase will not be considered.
+(Note: next-line ignore comments takes precedence over in-line ignores, so try to only use one for any given line!)
+
 ```bash
 This line has RULE_NAME but will be ignored # wokeignore:rule=RULE_NAME
 
-# for example, to ignore the following line for the whitelist rule
+# wokeignore:rule=RULENAME
+Here is another line with RULE_NAME that will be ignored
+
+# a couple of examples ignoring the following line for the whitelist rule
 whitelist # wokeignore:rule=whitelist
 
-# or for multiple rules
+# wokeignore:rule=whitelist
+whitelist
+
+# a couple of examples doing the same for multiple rules
 whitelist and blacklist # wokeignore:rule=whitelist,blacklist
+
+# wokeignore:rule=whitelist,blacklist
```

Comment:
suggestion: it might be worth mentioning/calling out that the rules list should be comma delimitated with no space characters.

Reply:
while the original documentation also doesn't include this note (since it would also apply to in-line ignores), I agree with you and will add this to both

Reply:
Awesome! I agree it wasn't a introduction you made into the codebase, but definitely a good contribution. You could write a simple note once somewhere. Doesn't have to be in multiple places.